src/components/OnboardingDialog/index.js

```diff
@@ -0,0 +1,139 @@
+import React, { useState } from 'react'
+import styled from '@emotion/styled'
+import PropTypes from 'prop-types'
+import { radius, shadow, space, duration, transition } from '../../theme'
+import { useTransition } from '../../hooks'
+
+const BodyWrapper = styled.div`
+  background-color: white;
+  border-radius: ${radius.lg};
+  box-shadow: ${shadow.strong};
+  padding: ${space[32]};
+`
+
+const OnboardingDialogBody = ({ children }) => (
+  <BodyWrapper>{children}</BodyWrapper>
+)
+
+OnboardingDialogBody.propTypes = {
+  children: PropTypes.oneOfType([
+    PropTypes.arrayOf(PropTypes.node),
+    PropTypes.node,
+  ]).isRequired,
+}
```

Comment:
I don't think we need propType validation for children 🤔

Reply:
Why not?

Reply:
Since it _is_ a prop, I think it does no harm to add it. Also, it will remind you if you forget to include it. What do you think, @Roohn?

Reply:
Well that is a good one, I never do it because the PropTypes are always the same.
But I see no harm in doing this. Maybe it will be good to do this in every component

Reply:
Yes let's do that!